src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+  width: 96px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-size: 16px 16px;

```

Comment:
Looks like your stars size and distance between them are different than than on mockup.
Check exact size if star image, star container dimensions in figma file.
![image](https://user-images.githubusercontent.com/60885495/125259174-bf60cb00-e307-11eb-9108-479fac54dae2.png)
![image](https://user-images.githubusercontent.com/60885495/125259218-c7206f80-e307-11eb-8162-6b7ecde556fb.png)